test/workflow/module_tests/test_cggnn.sh

```diff
@@ -0,0 +1,20 @@
+
+spt workflow configure --local --workflow='cggnn' --study-name='Melanoma intralesional IL2' --database-config-file=../db/.spt_db.config.container --workflow-config-file=module_tests/.workflow.config
+nextflow run .
+
+status=$?
+
+$([ $? -eq 0 ] && [ -e "results/mi2_model.pt" ] && [ -e "results/mi2_importances.csv" ])
+status="$?"
+
+head "results/mi2_importances.csv"
```

Comment:
I think we should be actually testing something about the output or result of doing this workflow. Either checking exact contents of the model binary, or checking a few values, etc. This is how most of the other tests are structured. The idea is that they should fail if the computation takes places in a different way. (And test artifacts would need updating if this failure is due to deliberate changes.)

Reply:
Earlier I missed that line 7 above does technically check that certain output files are created.
I still think we should check something about about the contents, e.g. the set of importances score values (rounded to 2 decimal or something like that).